community/primitive-collections/src/main/java/org/neo4j/concurrent/BinaryLatch.java

```diff
@@ -90,6 +90,18 @@ private void unparkSuccessor( Node waiters )
      */
     public void await()
     {
+        await( this );
+    }
+
+    /**
+     * Wait for the latch to be released, blocking the current thread with the specified blocker, if necessary.
+     *
+     * This method returns immediately if the latch has already been released.
+     * @param blocker The blocker that is given as the reason for blocking the thread, accessible through
+     * {@link java.util.concurrent.locks.LockSupport#getBlocker(Thread)}.
+     */
+    public void await( Object blocker )
```

Comment:
Yeah, it was introduced to support blocking methods in OptiLock, but then those blocking methods got removed and this guy remained. I'll remove it.